api/applications/applicationRouter.js

```diff
@@ -353,27 +361,67 @@ router.post(
  *        $ref: '#/components/responses/UnauthorizedError'
  */
 
-// registers a user with okta and approves their application ticket
+// Author: Christwide Oscar
+// Finding Applicant and setting them to approved
+router.post('/approve/:profile_id', (req) => {
+  const { profile_id, low_income } = req.body;
+  if (low_income === false || low_income === true) {
+    axios
+      .post(`${baseURL}/update/mentee/${profile_id}`, {
+        validate_status: 'approved',
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  } else {
+    axios
+      .post(`${baseURL}/update/mentor/${profile_id}`, {
+        validate_status: 'approved',
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+});
 
-router.put(
-  '/approve/:id',
-  authRequired,
-  adminRequired,
-  checkApplicationExists,
-  findProfile,
-  registerOktaUser,
-  (req, res, next) => {
-    const application_id = req.application.application_id;
+// Author: Farhaan Nishtar
+// Finding Applicant and setting them to rejected
+router.post('/reject/:profile_id', (req) => {
+  const { profile_id, low_income } = req.body;
+  if (low_income === false || low_income === true) {
```

Comment:
Perhaps an opportunity for `if  (typeof profile_id !== undefined)`. Just trying to think of a way to clean this up a little more.

Reply:
Hey John, that is actually a great Idea. I'll test it out and implement it.